plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/standard/StandardDeployCommandHandler.java

```diff
@@ -142,16 +140,23 @@ public void done(IJobChangeEvent event) {
     consoleManager.showConsoleView(messageConsole);
   }
 
-  private String getConsoleName(String project) {
+  private static String getConsoleName(String project) {
     Date now = new Date();
-    String nowString = DateFormat.getDateTimeInstance(
-        DateFormat.MEDIUM, DateFormat.MEDIUM, Locale.getDefault()).format(now);
-    return MessageFormat.format("{0} - {1} ({2})", CONSOLE_NAME, project, nowString);
+    String nowString = DateFormat.getDateTimeInstance(DateFormat.MEDIUM,
+                                                      DateFormat.MEDIUM,
+                                                      Locale.getDefault())
+                                 .format(now);
+    return MessageFormat.format("{0} - {1} ({2})",
+                                Messages.getString("deploy.console.name"),
+                                project,
+                                nowString);
   }
 
-  private StandardDeployJobConfig getDeployJobConfig(IProject project, Credential credential,
-      IPath workDirectory, MessageConsoleStream outputStream,
-      DefaultDeployConfiguration deployConfiguration) {
+  private static StandardDeployJobConfig createDeployJobConfig(IProject project,
+                                                               Credential credential,
+                                                               IPath workDirectory,
+                                                               MessageConsoleStream outputStream,
```

Comment:
So I believe you should probably create two streams, one for stdout and the other for stderr (with a different colour).  You can see the `org.eclipse.debug.ui.console.ConsoleColorProvider` for an example of setting colours. (We should punt this to a different issue.)